tests/cypress/integration/locale_selector_spec.js

```diff
@@ -1,34 +1,88 @@
-describe('Locale is set to ja-JP', () => {
+describe('Locale switching for H.O.C. home page', () => {
 
-  it('when Japanese is selected', () => {
+  it('works as expected', () => {
```

Comment:
same comment here, `works as expected` does not tell me what this test is about. Something like `navigates to a url with the correct language` will be better

Another option for this spec is to have one describe and several tests in it, like
```
describe('Locale switching`, () => {
  it('for HOC home page, () => {}
  it('for ASA', () => {}
  ...
}
```